modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserAccountResourceImpl.java

```diff
@@ -1082,6 +1082,12 @@ public UserAccount putUserAccountByExternalReferenceCode(
 			_getServiceBuilderPhones(null, userAccount),
 			_getWebsites(null, userAccount), false, serviceContext);
 
+		byte[] portraitBytes = _getPortraitBytes(true, user, userAccount);
+
+		if (ArrayUtil.isNotEmpty(portraitBytes)) {
+			_userService.updatePortrait(user.getUserId(), portraitBytes);
```

Comment:
here I'm pretty sure that we need to save it in the user variable because we are using it below to update the status